benchmark/jsonmask-java-benchmark/README.md

```diff
@@ -0,0 +1,32 @@
+# Overview
+This subproject contains performance measurement for [json-mask-java]() library.
```

Comment:
I also think that `jsonmask-java-benchmark` is unnecessary directory level. We could put everything into `benchmark`.